app/web/markdown/open-source.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Open Source & Sponsors
+crumb: Open Source & Sponsors
+type: general
+description: We want to give a huge shout out to the awesome open-source projects & technologies we build on, as well as thank the companies that sponsor us!
+---
+
+The driving force of Couchers is the global [volunteering team](/team), and they are the stars of the show.
+
+That said, Couchers is an open-source project and we are very heavy users of open-source technologies ourselves. We want to give a huge shout out to the folks who have built the tools and technologies that make Couchers possible! We are also supported by some great companies that help us build and operate Couchers, whether it be because we are a non-profit or an open-source project.
```

Comment:
```suggestion
That said, Couchers is an open-source project and we are very heavy users of open-source technologies ourselves. We want to give a huge shout out to the folks who have built the tools and technologies that make Couchers possible! We are also supported by some great companies that help us build and operate Couchers, whether it is because we are a non-profit or an open-source project.
```